### DISCUSSION TITLE
[Help]: How to disable Draggable property of slide

### DISCUSSION THREAD

**Discussion by archaditya** [davidjerleke/embla-carousel#1003] [Help] [answered]
### Summary

I need to disable this mouseScroll feature because it is stopping the auto play of slides. How can i disable the mouseScroll please help.

https://github.com/user-attachments/assets/d04e957e-572b-423e-8e29-4a38fefe0528

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment]
> How can i disable the mouseScroll please help

Hi @adityakkpk,
What do you mean by **mouseScroll**? Do you mean if it's draggable or not?

### On a side note
Please don't leave the title empty for this discussion.

**Reply by archaditya** [reply]
Yes, It is draggable but I want to disable the slide's draggable property when auto play is on. Because when the slides are sliding automatically and If I drag any slide then slides are not moving automatically.

and sorry , I miss the title.

**Reply by davidjerleke** [reply, marked as answer]
@adityakkpk use the `stopOnInteraction: false` option on the Autoplay plugin. Here's [how you change options](https://www.embla-carousel.com/api/plugins/#constructor-options) on plugins.

**Reply by archaditya** [reply]
Thanks @davidjerleke for the help.
